com.hazelcast.jet.server.JetCommandLineTest.test_cancelJob_byJobId

_master_ (commit f8276874c6b6fa7433e41a51f4ab885c9a40188c)

Failed on Oracle JDK 8: http://jenkins.hazelcast.com/view/Jet%20-%20Compatibility/job/jet-oss-master-oracle-jdk8/266/testReport/com.hazelcast.jet.server/JetCommandLineTest/test_cancelJob_byJobId/

Stacktrace:
```
picocli.CommandLine$ExecutionException: Error while calling command (public void com.hazelcast.jet.server.JetCommandLine.cancel(com.hazelcast.jet.server.JetCommandLine$Verbosity,com.hazelcast.jet.server.JetCommandLine$TargetsMixin,java.lang.String) throws java.io.IOException): com.hazelcast.core.HazelcastException: com.hazelcast.core.HazelcastException: Could not load 'hazelcast-client.yaml' from the classpath
	at picocli.CommandLine.execute(CommandLine.java:1207)
	at picocli.CommandLine.access$800(CommandLine.java:141)
	at picocli.CommandLine$RunAll.handle(CommandLine.java:1416)
	at picocli.CommandLine$RunAll.handle(CommandLine.java:1376)
	at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:1243)
	at picocli.CommandLine.parseWithHandlers(CommandLine.java:1526)
	at com.hazelcast.jet.server.JetCommandLine.runCommandLine(JetCommandLine.java:162)
	at com.hazelcast.jet.server.JetCommandLineTest.run(JetCommandLineTest.java:666)
	at com.hazelcast.jet.server.JetCommandLineTest.test_cancelJob_byJobId(JetCommandLineTest.java:200)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.hazelcast.core.HazelcastException: com.hazelcast.core.HazelcastException: Could not load 'hazelcast-client.yaml' from the classpath
	at com.hazelcast.internal.config.AbstractConfigLocator.loadConfigurationFromClasspath(AbstractConfigLocator.java:164)
	at com.hazelcast.internal.config.AbstractConfigLocator.loadConfigurationFromClasspath(AbstractConfigLocator.java:176)
	at com.hazelcast.jet.impl.config.YamlJetClientConfigLocator.locateOnClasspath(YamlJetClientConfigLocator.java:52)
	at com.hazelcast.internal.config.AbstractConfigLocator.locateInWorkDirOrOnClasspath(AbstractConfigLocator.java:124)
	at com.hazelcast.jet.impl.config.ConfigProvider.locateAndGetClientConfig(ConfigProvider.java:101)
	at com.hazelcast.jet.server.JetCommandLine.getClientConfig(JetCommandLine.java:481)
	at com.hazelcast.jet.server.JetCommandLine.lambda$getJetClient$13(JetCommandLine.java:463)
	at com.hazelcast.jet.impl.util.Util.uncheckCall(Util.java:92)
	at com.hazelcast.jet.server.JetCommandLine.getJetClient(JetCommandLine.java:463)
	at com.hazelcast.jet.server.JetCommandLine.runWithJet(JetCommandLine.java:454)
	at com.hazelcast.jet.server.JetCommandLine.cancel(JetCommandLine.java:253)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at picocli.CommandLine.execute(CommandLine.java:1189)
	... 20 more
Caused by: com.hazelcast.core.HazelcastException: Could not load 'hazelcast-client.yaml' from the classpath
	at com.hazelcast.internal.config.AbstractConfigLocator.loadConfigurationFromClasspath(AbstractConfigLocator.java:160)
	... 35 more
```

Standard output:
```
Hiccups measured while running test 'test_cancelJob_byJobId(com.hazelcast.jet.server.JetCommandLineTest):'
11:51:05, accumulated pauses: 1221 ms, max pause: 340 ms, pauses over 1000 ms: 0
11:51:10, accumulated pauses: 3348 ms, max pause: 253 ms, pauses over 1000 ms: 0
11:51:15, accumulated pauses: 3275 ms, max pause: 423 ms, pauses over 1000 ms: 0
11:51:20, accumulated pauses: 4523 ms, max pause: 890 ms, pauses over 1000 ms: 0
11:51:25, accumulated pauses: 2662 ms, max pause: 264 ms, pauses over 1000 ms: 0
11:51:30, accumulated pauses: 143 ms, max pause: 40 ms, pauses over 1000 ms: 0
```

--- a/hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java
+++ b/hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java
@@ -30,7 +30,7 @@ import com.hazelcast.jet.pipeline.Pipeline;
 import com.hazelcast.jet.pipeline.Sinks;
 import com.hazelcast.jet.pipeline.Sources;
 import com.hazelcast.map.IMap;
-import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastSerialClassRunner;
 import org.junit.After;
 import org.junit.AfterClass;
 import org.junit.Before;
@@ -61,7 +61,7 @@ import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
-@RunWith(HazelcastParallelClassRunner.class)
+@RunWith(HazelcastSerialClassRunner.class)
 public class JetCommandLineTest extends JetTestSupport {
 
     private static final String SOURCE_NAME = "source";

